FileProcessor/src/main/java/com/group7/FileProcessor/entries/RemFile.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
+ */
+package com.group7.FileProcessor.entries;
+
+/**
+ *
+ * @author shale
+ */
+public class RemFile extends LocFile{
+    
+    String accessKey;
+    String repositoryId;
+    
+    public RemFile(String address,String accessKey,String repositoryId){
```

Comment:
Remove the address from the constructor. We won't have this when we construct this object cause all we know it that its a remote entry. My recommendation would be to accutally create a new folder on disk (with the same name as the remote entry) and then pass the file path of the newly creates file as the address that gets passed to the super constructor. Its a little scuffed but I think it could work.